projects/05-wordpress-plugin/README.md

```diff
@@ -0,0 +1,254 @@
+# Plugin para Wordpress
+
+## Índice
+
+* [1. Preámbulo](#1-preámbulo)
+* [2. Resumen del proyecto](#2-resumen-del-proyecto)
+* [3. Objetivos de aprendizaje](#3-objetivos-de-aprendizaje)
+* [4. Consideraciones generales](#4-consideraciones-generales)
+* [5. Criterios de aceptación mínimos del proyecto](#5-criterios-de-aceptación-mínimos-del-proyecto)
+* [6. Pistas, tips y lecturas complementarias](#6-pistas-tips-y-lecturas-complementarias)
+
+***
+
+## 1. Preámbulo
+
+> Los sistemas de gestión de contenidos o CMS (Content Management System)  son
+> aplicaciones web que permiten a usuarios (sin necesidad de tener conocimientos
+> previos en programación)  crear contenidos con una interfaz visual de manera
+> rápida y sencilla. Entre los más utilizados están [Wordpress(42.1%),
+> Shopify(3.7%), Joomla(2.0%), Squarespace(1.7%) y
+> Wix(1.7%)](https://w3techs.com/technologies/overview/content_management).
+>
+> Wordpress es uno de los CMS más utilizados en el mercado, éste permite crear
```

Comment:
```suggestion
> Wordpress es uno de los CMS más utilizados en el mercado. Éste permite crear
```

Reply:
Al ser la primera mención de Wordpress, quizás también haría que la palabra _Wordpress_ sea un link a la página oficial.

Reply:
También me acabo de dar cuenta de que en las páginas _oficiales_ siempre usan _WordPress_ en camel case. Propongo hacer un find-and-replace para asegurarnos de que todas las menciones a WordPress usen esa convención. Ahora a veces aparece como Wordpress y otras como WordPress.